### ISSUE TITLE
Shouldn't you use | for XORing flags and not &?

### ISSUE THREAD

**Issue by Keithcat1** [Timtam/Bass4Py#13]
I was under the impression that & is used to check the presence of a flag while | is used to toggle a flag.
For example bass4py/bass/stream.pyx:line 89

**Comment by Timtam** [comment]
Yeah, you're right, that's how its supposed to be. I however don't find a line like that in stream.pyx:89. Mind pulling the latest version and giving me the probably errorneous line afterwards or just telling me its content so that I can verify thats actually wrong? I literally reworked that new Cython version over 3 times during the last 7 years so there are alot of errors which I didn't catch or introduced myself by reworking it so many times. That might be one of them.

**Comment by Keithcat1** [comment]
That's because I was being a potato and giving the wrong filename by accident.

bass4py/bass/syncs.pyx:line 89
```cython
      type = type & _BASS_SYNC_ONETIME
```

**Comment by Timtam** [comment]
yep, that seems to be an error for sure. I'll have to develop the trait to always write tests whenever I implement a new functionality, 'cause that would have been catched exactly by tests I guess. Its hard to break with habits though.
